README.md

```diff
@@ -296,12 +306,17 @@ linter within the project's directory tree, it will enforce a line length limit
 of 100 instead of 80. This will also be true for editors which rely on either
 linter.
 
+All user-facing scripts have been linted with `shellcheck` for common
```

Comment:
Can you describe here how to run this linter so that as people change scripts they can re-lint them?